usaspending_api/disaster/management/commands/generate_covid19_download.py

```diff
@@ -41,7 +41,42 @@ class Command(BaseCommand):
     readme_path = Path(settings.COVID19_DOWNLOAD_README_FILE_PATH)
     full_timestamp = datetime.strftime(datetime.now(timezone.utc), "%Y-%m-%d_H%HM%MS%S%f")
 
+    # KEY is the type of compute supported by this command
+    # key's VALUE are the strategies required by the compute type
+    # These strategies are used to change the behavior of this command
+    #   at runtime.
+    compute_flavors = {
+        ComputeFlavorEnum.AURORA.value: {
```

Comment:
Usually when we refer to a database version of something in code, we call it Postgres not Aurora. If we were to run this locally for instance, we wouldn't be running it in Aurora, we'd be running it on a local Postgres instance.

Reply:
Ok so you're saying you'd like me to change this value to postgres?

Reply:
Done